hdbscan/validity.py

```diff
@@ -180,8 +180,8 @@ def internal_minimum_spanning_tree(mr_distances):
     # A little "fancy" we select from the flattened array reshape back
     # (Fortran format to get indexing right) and take the product to do an and
     # then convert back to boolean type.
-    edge_selection = np.prod(np.in1d(min_span_tree.T[:2], vertices).reshape(
-        (min_span_tree.shape[0], 2), order='F'), axis=1).astype(bool)
+    edge_selection = np.prod(
+        np.isin(min_span_tree.T[:2], vertices), axis=0).astype(bool)
```

Comment:
Seems correct, but may warrent a look.  Anything else doesn't make sense to me anyway, the `order="F"` was breaking things when switching to `isin`, which doesn't flatten the result (keeps shape of `vertices`).

The other places want to work with 1-D arrays as far as I can tell, so this distinction doesn't matter.

Reply:
Yes, this should be correct. ``isin`` is a much nicer way to do this, but didn't exist back when this was first written.